js/graviex.js

```diff
@@ -0,0 +1,231 @@
+'use strict';
+
+// ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+// const { AuthenticationError, InsufficientFunds } = require ('./base/errors');
+// const Precise = require ('./base/Precise');
+
+// ---------------------------------------------------------------------------
+
+module.exports = class graviex extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'graviex',
+            'name': 'Graviex',
+            'version': 'v3',
+            'countries': [ 'MT', 'RU' ],
+            'rateLimit': 1000,
+            'has': {
+                'CORS': undefined,
+                'fetchMarkets': true,
+                'fetchTicker': true,
+                'fetchTickers': true,
+                'fetchBalance': false,
+                'fetchCurrencies': false,
+                'fetchTrades': false,
+                'fetchOHLCV': false,
+                'fetchOrders': false,
+                'fetchOrder': false,
+                'fetchOrderBook': false,
+                'fetchOrderBooks': false,
+                'fetchL2OrderBook': false,
+                'fetchOrderTrades': false,
+                'fetchMyTrades': false,
+                'fetchWithdrawals': false,
+                'fetchDeposits': false,
+                'fetchDepositAddress': false,
+                'createOrder': false,
+                'cancelOrder': false,
+                'createOcoOrder': false,
+            },
+            'timeframes': {
+                '1m': '1',
+                '5m': '5',
+                '15m': '15',
+                '30m': '30',
+                '1h': '60',
+                '2h': '120',
+                '4h': '240',
+                '6h': '360',
+                '12h': '720',
+                '1d': '1440',
+                '3d': '4320',
+                '1w': '10080',
+            },
+            'urls': {
+                'logo': '',
+                'api': {
+                    'public': 'https://graviex.net//webapi/v3',
+                    'private': 'https://graviex.net//webapi/v3',
+                },
+                'www': 'https://graviex.net',
+                'doc': 'https://graviex.net/documents/api_v3',
+                'fees': 'https://graviex.net/documents/fees-and-discounts',
+            },
+            'api': {
+                // market: Unique market id. It's always in the form of xxxyyy, where xxx is the base currency code, yyy is the quote currency code, e.g. 'btccny'. All available markets can be found at /api/v2/markets.
+                'public': {
+                    'get': [
+                        'markets',
+                        'tickers',
+                        'trades', // Requires market XXXYYY. Optional: limit, timestamp, from, to, order_by
```

Comment:
We will need to list out all their public and private endpoints here.

Reply:
OK. Fixed